exporting/tests/exporting_doubles.c

```diff
@@ -16,13 +16,11 @@ struct engine *__mock_read_exporting_config()
     engine->config.hostname = strdupz("test-host");
     engine->config.update_every = 3;
 
-    engine->connector_root = calloc(1, sizeof(struct connector));
-    engine->connector_root->config.type = BACKEND_TYPE_GRAPHITE;
-    engine->connector_root->engine = engine;
 
-    engine->connector_root->instance_root = calloc(1, sizeof(struct instance));
-    struct instance *instance = engine->connector_root->instance_root;
-    instance->connector = engine->connector_root;
+    engine->instance_root = calloc(1, sizeof(struct instance));
```

Comment:
Considering that this is a test, we do not need the `callocz`, but I believe we need to test the result.

Reply:
I don't think we need to handle all possible errors in tests. If the test crashes we can trace our code.